exercises/concept/lucians-luscious-lasagna/.docs/instructions.md

```diff
@@ -2,7 +2,7 @@
 
 In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
 
-You have four tasks, all related to the time spent cooking the lasagna.
+You have four tasks, all related to the time spent cooking the lasagna. Use `int` as type for inputs and output.
```

Comment:
```suggestion
You have four tasks, all related to the time spent cooking the lasagna.

~~~~exercism/note
You can either use type inference, where you don't specify any types, or use explicit typing, where you do specify the types.
If you choose the latter, use `int` as the type.
~~~~
```

Reply:
@s-praveen-kumar What do you think about the above suggestion?

Reply:
Yes @ErikSchierboom . This is much clearer